Vostok.Metrics.AspNetCore/Helpers/EventHelper.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+
+namespace Vostok.Metrics.AspNetCore.Helpers;
+
+internal static class EventHelper
```

Comment:
This class can probably be moved to `Vostok.Commons.Helpers` so that it is reused in `Vostok.Metrics.System`, if that's appropriate. It's up to you on this, though.

Reply:
https://github.com/vostok/metrics.system/pull/33